### DISCUSSION TITLE
Confusing version numbers

### DISCUSSION THREAD

**Discussion by AHorak** [react-hook-form/react-hook-form#2748] [General] [answered]
The product is developed very quickly.
I'm not quite clear about the version numbers and compatibility.
Is 6.6.X incompatible with 6.5.X? When I look at the list, it looks more like bugfixes for me. Wouldn't you rather count up the last version number?

**Comment by bluebill1049** [comment, marked as answer]
have a look here: https://semver.org/ we follow semantic versioning.

**Reply by bluebill1049** [reply]
yes 6.6.0 is compatible with 6.5.0

**Reply by AHorak** [reply]
Thank you :-)

**Reply by AHorak** [reply]
> 
> 
> have a look here: https://semver.org/ we follow semantic versioning.

Thank you for the Link.  I was not able to recognize, if the changes in 6.6.0 were  "functionality in a backwards compatible manner" or bugfixes :-)

**Reply by bluebill1049** [reply]
functionality in a backwards compatible manner plus bugfixed. take a look at the changelog for what's changed.
